src/org/omegat/logger.properties

```diff
@@ -8,16 +8,19 @@ org.omegat.level = ALL
 java.util.logging.ConsoleHandler.level = ALL
 org.omegat.util.logging.OmegaTFileHandler.level = ALL
 
-org.omegat.util.logging.OmegaTLogFormatter.mask=$mark: $level: $text $key
+org.omegat.util.logging.OmegaTLogFormatter.mask=$time: $level: $text $key
 #org.omegat.util.logging.OmegaTLogFormatter.mask=$time: $threadName [$level] $key $text
+#org.omegat.util.logging.OmegaTLogFormatter.mask=$mark: $level: $text $key
 
-org.omegat.util.logging.OmegaTLogFormatter.timeFormat=HH:mm:ss,SSSS
+org.omegat.util.logging.OmegaTLogFormatter.timeFormat=HH:mm:ss.SSS
 
 java.util.logging.ConsoleHandler.formatter = org.omegat.util.logging.OmegaTLogFormatter
 
 org.omegat.util.logging.OmegaTFileHandler.formatter = org.omegat.util.logging.OmegaTLogFormatter
 org.omegat.util.logging.OmegaTFileHandler.size=1048576
```

Comment:
Are these lines still useful if we change the retention policy? What happens if the file os over 1 MB?

Reply:
These two property values are used in removed `rotate` method in the handler. Now it is not used.

Reply:
adjust it by removing two property in the file.

Reply:
We can respect `count` property when removing old files, we always keep `count` number of recent files, even if retention date passed.

Reply:
When respecting `count`, it is not good for file name convention because it is not able to sort of creation date by file name, which consist of session id first.

Now I would like to propose a change of file name format to be `OmegaT_yyyyMMdd-hhmmss_sessionId.log`

Reply:
Good idea, I agree

Reply:
> Now I would like to propose a change of file name format to be OmegaT_yyyyMMdd-hhmmss_sessionId.log

For me the best retention policy would be based on the size of the directory, but I suppose it is harder to implement?

Reply:
> For me the best retention policy would be based on the size of the directory, but I suppose it is harder to implement?

I don't have an idea how to implement it. You are welcome to propose new  method to do it by a new property.

Reply:
Which behavior is more natural?

1. always keep minimum 10 log files and then retention based on time
2. retention based on file date but maximum files to be kept is 10